Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -18,25 +18,40 @@ namespace Microsoft.Toolkit.Uwp.Helpers
     /// <summary>
     /// This class provides info about the app and the system.
     /// </summary>
-    public static class SystemInformation
+    public class SystemInformation
```

Comment:
```suggestion
    public sealed class SystemInformation
```

Shouldn't this class be `sealed`? This should also help the JIT/.NET Native compiler resolve and inline the `callvirt` opcodes when accessing the property getters, which could help performance a bit.

Reply:
Yes, I've added the sealed attribute on the class.